README.md

```diff
@@ -39,14 +39,20 @@ pipx install beets
 pipx inject beets beetcamp [python-mpd2 ...]
 ```
 
+If the CLI is not available after injecting, uninject and try again using the `--include-apps` flag.
+
+```bash
+pipx inject --include-apps beets beetcamp
```

Comment:
I think we may as well just include this flag in the existing `pipx inject` instruction on line 39: we can be certain that `beetcamp` won't be made available unless `--include-apps` is provided.

Consider extending the description with something like 

... _`--include-apps` flag is required to make sure that `beetcamp` is made available in your command line_ ...